src/main/scala/org/bblfsh/client/v2/Node.scala

```diff
@@ -0,0 +1,5 @@
+package org.bblfsh.client.v2
+	
+case class Node(ctx: Long, handle: Long) {
+  @native def load(): Map[String, _]
```

Comment:
I'm not sure it will return `Map[string, _]` in all cases. The node may be a single string value or an array, for example.

Reply:
Good catch, thank you, will add a comment not to forget when working on `local()`

Reply:
This also means that we can not use https://github.com/bblfsh/client-scala/issues/83#issuecomment-494245514 as we are forced to have some equivalent of `Node` and `NodeExt` distinction, to represent 1) something in JVM that can be `value, array or map`  and  2) a mere proxy to native node.

Reply:
I don't think it's practical to allow to load maps only. We'll need to find a way to make it work for other types as well.

Reply:
Agreed (was just exploring all possible options).
Will introduce a proper Node abstraction (and something like NodeExt separation) in next PR.